scene/resources/curve.cpp

```diff
@@ -49,6 +49,19 @@ const char *Curve::SIGNAL_RANGE_CHANGED = "range_changed";
 Curve::Curve() {
 }
 
+void Curve::set_point_count(int p_count) {
+	ERR_FAIL_COND(p_count < 0);
+	if (_points.size() >= p_count) {
+		_points.resize(p_count);
+		mark_dirty();
+	} else {
+		int i = p_count - _points.size();
+		for (; i > 0; i--) {
```

Comment:
Why not this?
```suggestion
		for (int i = p_count - _points.size(); i > 0; i--) {
```

Reply:
I'm ok with that, to be honest I don't remember why I did it like that.